config.pri

```diff
@@ -14,11 +14,10 @@ QT_VER_MAJ = $$member(QT_VERSION, 0)
 QT_VER_MIN = $$member(QT_VERSION, 1)
 
 lessThan(QT_VER_MAJ, 5) {
-error(YACReader requires Qt 5 or newer but Qt $$[QT_VERSION] was detected.)
-  }
 lessThan(QT_VER_MIN, 6):!CONFIG(no_opengl) {
   error ("You need at least Qt 5.6 to compile YACReader or YACReaderLibrary.")
   }
+}
```

Comment:
I think this change is incorrect. The `lessThan(QT_VER_MAJ, 5)` should be restored to reject Qt 4. And the `lessThan(QT_VER_MIN, 6)` should be enclosed in a separate `equals(QT_VER_MAJ, 5)` block.

Reply:
Absolutely